addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.subtitles.torec"
        name="Torec"
-       version="1.3.1"
+       version="1.4.2"
```

Comment:
there's no point in bumping up version, until published - so change to `1.4.0` here